WireFoundation/Package.swift

```diff
@@ -7,6 +7,8 @@ let package = Package(
     name: "WireFoundation",
     platforms: [.iOS(.v16), .macOS(.v12)],
     products: [
+        .library(name: "Clibsodium", targets: ["Clibsodium"]),
```

Comment:
We can remove this as a product once we remove `WireCryptobox`

Reply:
might be good to left this comment here as a 
```suggestion
        // TODO: [WPB-7394] remove libSodium not needed anymore
        .library(name: "Clibsodium", targets: ["Clibsodium"]),
```

(https://wearezeta.atlassian.net/browse/WPB-7394)]